### DISCUSSION TITLE
Welcome to Semantic Kernel discussions!

### DISCUSSION THREAD

**Discussion by dluc** [microsoft/semantic-kernel#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below.

**Comment by garzaroland95** [comment]
Hello. I'm new to this, if anyone can give me any beginner advice or pointer's. I'd really appreciate it. Thanks

**Comment by Callmebhavani** [comment]
Sir i want to practice the hacking

**Comment by bgarnr** [comment]
I have a .net core solution and was running tests using large json documents as part of my system messages to chatgpt to maintain application state and context. I had started looking at using pinecone for persisting my 10k token document between api calls but see there is a qdrant sample. Is qdrant an appropriate approach?

**Comment by imranshams** [comment]
At our company, we invest a significant amount of time creating individual agents using the semantic kernel. Your efforts in this area are commendable.

To ensure we can deliver our agents with the necessary SLA, it would be extremely helpful if you could share the semantic kernel roadmap for this year. Given the numerous changes taking place, having this information will greatly assist us in planning and maintaining our service standards.

Thank you.
